modules/tinymce/src/core/main/ts/api/Editor.ts

```diff
@@ -1015,8 +1026,14 @@ class Editor implements EditorObservable {
       return urlConverterCallback.call(self, url, elm, true, name);
     }
 
-    // Don't convert link href since thats the CSS files that gets loaded into the editor also skip local file URLs
-    if (!getOption('convert_urls') || (elm && elm.nodeName === 'LINK') || url.indexOf('file:') === 0 || url.length === 0) {
+    // Don't convert link href since that's the CSS files that gets loaded into the editor also skip local file URLs
+    if (
+      !getOption('convert_urls') ||
+      elm === 'link' ||
```

Comment:
Code change here as `elm` can be a string or an Element and this only accounted for the element path.